app/controllers/concerns/hal_actions.rb

```diff
@@ -16,14 +16,14 @@ def show
   end
 
   def create
-    self.resource = consume!(resource)
+    consume_with_content_type! resource
```

Comment:
That is, you have to hand it an object (like `Story.new`) which it then modifies. `resource`, in this case, is our object which is modified.